src/main/webapp/new/app/shared/domain/game-map/edge.ts

```diff
@@ -80,6 +80,17 @@ export class Edge {
         return null;
     }
 
+    isJoint() {
+        let hexesCount = 0;
+        for (let i in this.hexes) {
+            hexesCount++;
+            if (hexesCount > 1) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
but "hexes" is not an array, it's an object, which has 6 properties, which can be null or have a value. We can add method size() which calculate count, but it will look like same as "isJoint()"